packages/mdc-drawer/slidable/_mixins.css

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "@material/rtl/mixins";
```

Comment:
Honestly...I have no idea, they were existing magic numbers in temporary drawers.

I've isolated them to slideable's _mixins.scss in this Pull Request. I think a follow up pull request might want to investigate these magic numbers more